source/postprocess/velocity_statistics.cc

```diff
@@ -35,34 +35,56 @@ namespace aspect
     std::pair<std::string,std::string>
     VelocityStatistics<dim>::execute (TableHandler &statistics)
     {
-      const Quadrature<dim> &quadrature_formula = this->introspection().quadratures.velocities;
-      const unsigned int n_q_points = quadrature_formula.size();
-
-      FEValues<dim> fe_values (this->get_mapping(),
-                               this->get_fe(),
-                               quadrature_formula,
-                               update_values   |
-                               update_quadrature_points |
-                               update_JxW_values);
-      std::vector<Tensor<1,dim>> velocity_values(n_q_points);
+      const QIterated<dim> quadrature_formula_trapz (QTrapezoid<1>(),
+                                                     this->get_parameters().stokes_velocity_degree);
+      const unsigned int n_q_trapz_points = quadrature_formula_trapz.size();
+
+      const Quadrature<dim> &quadrature_formula_face = this->introspection().quadratures.velocities;
```

Comment:
```suggestion
      const Quadrature<dim> &quadrature_formula_face = this->introspection().quadratures.velocities;
```
face is not a good name, maybe ``_for_rms``?